src/PolarSSL++/SslContext.h

```diff
@@ -127,13 +127,13 @@ class cSslContext abstract
 	/** The callback used by PolarSSL when it wants to read encrypted data. */
 	static int ReceiveEncrypted(void * a_This, unsigned char * a_Buffer, size_t a_NumBytes)
 	{
-		return ((cSslContext *)a_This)->ReceiveEncrypted(a_Buffer, a_NumBytes);
+		return (static_cast<cSslContext *>(a_This))->ReceiveEncrypted(a_Buffer, a_NumBytes);
```

Comment:
Actually it seems to have been explicitly allowed in C++11: http://www.open-std.org/jtc1/sc22/wg21/docs/cwg_defects.html#1120